app/constants/roles.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  STAFF: 'Staff',
+  MANAGER: 'Manager',
+  DATA_ADMIN: 'Data Admin',
+  SYSTEM_ADMIN: 'System Admin'
```

Comment:
Yes, these are to be stored in DB. They are reference data in the roles table. I think they should be left as they are since they are used as a comparison for results from the DB.